<?xml version="1.0"?>
<launch>

   <arg name="rviz" default="true" />
   <arg name="config_name"    default="/config/kaist_data_clins.yaml" />

  <!-- <node pkg="clins" type="clins_offline_node" name="clins_offline_node"
         output="screen" clear_params="true" required="true" launch-prefix="gdb -ex run &#45;&#45;args">-->
   <node pkg="clins" type="clins_offline_node" name="clins_offline_node" output="screen" >
      <param name="config_name"         type="string"   value="$(arg config_name)" />
   </node>

   <group if="$(arg rviz)">
     <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz"
        args="-d $(find clins)/launch/odom.rviz" required="true"/>
   </group>

</launch>
